### PR TITLE
Problem: Support Multiple AuthInfo Fee Amount

### LINKED ISSUE #272: Problem: Support multiple authInfo `Fee` objects

### PR DESCRIPTION
**Pull request by cdc-Hitesh**
Fixes #272

### CODE REVIEW

**Review by calvinaco — CHANGES_REQUESTED** [review]
The code looks good to me, but since this change affect tx construction, please add unit test and e2e tests for multiple coin cases. Thank you

File: lib/src/transaction/signable.ts [hunk 1]
```diff
@@ -145,28 +147,23 @@ export class SignableTransaction {
...
-        if (feeAmount) {
-            const feeAmountString = feeAmount.amount!;
+        cosmosAuthInfo.fee.amount.forEach((feeAmount) => {
+            const feeAmountString = feeAmount.amount;
             const feeAmountDenom = feeAmount.denom;
-            feeAmountCoin = croSdk.Coin.fromCustomAmountDenom(feeAmountString, feeAmountDenom);
-        }
+            const feeAmountCoin = croSdk.Coin.fromCustomAmountDenom(feeAmountString, feeAmountDenom);
+            feeAmountList.push(feeAmountCoin);
+        });
```
Comment (calvinaco) [inline lib/src/transaction/signable.ts:161]: Consider using `.map`

Comment (cdc-Hitesh) [reply lib/src/transaction/signable.ts:161]: sure thanks

Comment (cdc-Hitesh) [reply lib/src/transaction/signable.ts:161]: Btw, Can I understand the benefit of using `.map` here? Or is it more of a general practice?

Comment (cdc-Hitesh) [reply lib/src/transaction/signable.ts:161]: @calvinaco

Comment (calvinaco) [reply lib/src/transaction/signable.ts:161]: `.forEach` is more for iterating elements, but in this example you are iterating it and then return a new array based on the values, which makes it a perfect use of `.map`.

At the end of the day both will works, but it is more intuitive to use `.map` in thsi case.

Comment (cdc-Hitesh) [reply lib/src/transaction/signable.ts:161]: I see. Got it. Using .map will even save me a line or two.

Comment (cdc-Hitesh) [reply lib/src/transaction/signable.ts]: Resolved.

File: lib/src/transaction/raw.ts [hunk 2]
```diff
@@ -160,19 +160,37 @@ export const rawTransaction = function (config: InitConfigurations) {
...
             return this;
         }
 
+        /**
+         * Appends an `Amount` to the AuthInfo Fee Amount List
+         * @param {ICoin} feeAmount to be set to the raw tx body
+         * @returns {RawTransaction}
+         * @throws {Error} when fee set is invalid
+         * @memberof Transaction
+         * @deprecated
```
Comment (calvinaco) [inline lib/src/transaction/raw.ts:183]: Is this supposed to be deprecated?

**Review by calvinaco — APPROVED** [review]

File: lib/src/transaction/raw.ts (hunk 2 above)
Comment (cdc-Hitesh) [reply lib/src/transaction/raw.ts]: Good catch, 
Only `setFee` is supposed to be deprecated.

**Pull request merged**